test/units/testsuite-86.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -eux
+set -o pipefail
+
+mkdir -p /var/lib/machines/mymachine.raw.v
```

Comment:
Use /run or /tmp for ephemeral test files please

Reply:
also given this test doesn't require any particular dependency and doesn't seem particularly slow, it could be folded as a subtest of TEST-74-AUX-UTILS

Reply:
> Use /run or /tmp for ephemeral test files please

Doesn't work. the image discovery logic doesn't look there, and the test is supposed to test that

Reply:
There's no /run equivalent for machines? Ok, then can you mount a tmpfs on top of it for the test? That way we know nothing is left behind between runs

Reply:
CI tests 13 and 25 also add stuff to the dir. If we'd overmount it, then we'd fuck those up in case they are run in parallel with this one.

This integration test carefully removes the three dirs it puts there. It's not all over the place after all, it's all inside one of three .v/ dirs we put there. I think it's manageable to remove it again from there.

Reply:
It wouldn't affect other tests? They would be running under a different instance, not on the same kernel/namespace, so they wouldn't see this mount. We are trying to make sure the tests are all idempotent for the mkosi work, see: https://github.com/systemd/systemd/pull/30464

Reply:
also, this should really be folded as a subtest into 74-AUX-UTILS, there's no need to boot a separate image with all that overhead to test a local-only unprivileged tool

Reply:
it tests privileged stuff though: systemd-run -p RootDirectory= and systemd-dissect --discover.

Reply:
Sure that's ok, I meant to say in the sense that it doesn't need an image, so it doesn't need to be forced to run in qemu

Reply:
Still unsolved: needs a trap to cleanup var otherwise it will leave test artifacts behind on error, and should really be a subtest of test-74

Reply:
not really grokking why it is a problem to leave the resources around on failure, seems hard to debug if we delete everything that we failed with?

Reply:
The opposite, it gets harder, because an individual test run leaves state that could affect the next individual run. In the worst case, where some random element is used, then stuff even accumulates. So as a rule, all the integration tests do not leave stuff behind. This state is easy to recreate in case it's needed.

Reply:
i find test-74 super unusable? it's super heavy, are we sure we want to stuff even more stuff into that? makes it really hard to run tests if they take ages even on my machine?

Reply:
RTFM my dude! Assuming you add `testsuite-74.vpick.sh`, just do `make -C test/TEST-74-AUX-UTILS/ TEST_MATCH_SUBTEST=vpick` and ***magic*** will happen